source/includes/_dataset_status.md

```diff
@@ -19,7 +38,7 @@ Attribute | Description
 `published` <br> *boolean* | true if the dataset is available in the search API
 `name` <br> *string* | One of the dataset status values
 `since` <br> *datetime* | Timestamp when the dataset entered in the current status
-`user` <br> *user* <br> <em class="expandable">expandable</em> | User who started the action
+`user` <br> *user* <br> <em class="expandable">expandable</em> | User who started the action (currently not available in the API)
```

Comment:
Let's remove it entirely then, I really think the documentation should reflect the current state of the API, not a desired state.

Also, we need to add previous and next in this list since they are available.
And we need to add previous and next attributes to all examples in this section.

Reply:
As I understood it, "next" is not implemented yet, so let's not consider it in the current state of the API neither.